adoc/admin-disaster-recovery-backup.adoc

```diff
@@ -0,0 +1,149 @@
+= Backup
+This chapter describes the backup of {productname}.
+
+== Data To Backup
+* The `skuba`: command-line binary for the running cluster, use to replace nodes from cluster.
+* The skuba cluster folder: directory where used for running `skuba` commands. Contains cluster configurations and authentication files.
+* The kubernetes folder of every node in cluster: directory contains kubernetes configurations.
++
+```
+/etc/kubernetes
+```
+* The kubelet folder of every node in cluster: directory contains kubelet configurations.
++
+```
+/var/lib/kubelet
+```
+* The etcd cluster database: holds all cluster data. Can be used to recover master nodes.
+
+== Create Etcd Cluster Database Backup
+=== Procedure
+1. Mount external storage device to all master nodes. This is only required if following step is using local hostpath as volume storage.
+2. Create backup.
++
+Below is the sample manifest bind to local hostpath. Recommend to use other storage methods instead.
```

Comment:
It deserves mentioning below items.

1. have enough disk space
2. have user's backup retention policy